gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -1016,6 +1085,26 @@ _compare_tensors (GstTensorsInfo * info1, GstTensorsInfo * info2)
   return result;
 }
 
+/**
+ * @brief Copy GstTensorsInfo without the name of tensors.
+ * @param[out] the destination object
+ * @param[in] the source object
+ */
+static void
+gst_tensor_filter_copy_info (GstTensorsInfo * dest, GstTensorsInfo * src)
+{
+  guint i;
+
+  g_return_if_fail (dest != NULL && src != NULL);
+
+  dest->num_tensors = src->num_tensors;
+  for (i = 0; i < src->num_tensors; ++i) {
+    memcpy (dest->info[i].dimension, src->info[i].dimension,
+        sizeof (tensor_dim));
```

Comment:
Let's not loop here. you can call memcpy once on the whole src->info. (info[] size is constant, right?)

Reply:
Since we have to maintain `each dest->info[i].name`, we cannot use memcpy for the whole structure.